pkg/microservice/aslan/core/workflow/service/webhook/gitlab_workflow_task.go

```diff
@@ -335,6 +336,9 @@ func UpdateWorkflowTaskArgs(triggerYaml *TriggerYaml, workflow *commonmodels.Wor
 		moduleTest, err := commonrepo.NewTestingColl().Find(test.Name, workflow.ProductTmplName)
 		if err != nil {
 			log.Errorf("fail to find test TestModuleName:%s, workflowname:%s,productTmplName:%s,error:%v", test.Name, workflow.Name, workflow.ProductTmplName, err)
+			if err == mongo.ErrNoDocuments {
+				continue
+			}
```

Comment:
better not import mongo to service level

Reply:
the same to you